ftw/upgrade/testing.py

```diff
@@ -52,6 +52,17 @@ def setUpZope(self, app, configurationContext):
 
         z2.installProduct(app, 'Products.CMFPlacefulWorkflow')
 
+        try:
+            # Plone 4 with collective.indexing
+            pkg_resources.get_distribution('collective.indexing')
+
+            import collective.indexing
+            xmlconfig.file('configure.zcml', collective.indexing,
+                           context=configurationContext)
+            z2.installProduct(app, 'collective.indexing')
+        except pkg_resources.DistributionNotFound:
```

Comment:
👍  even though getting a `DistributionNotFound` may be unlikely here we should write the exception handling statements in a way that reduces the risk to swallow exceptions by accident. Also improves readability imho, i'll amend.